apps/desktop/src/lib/commit/CommitCard.svelte

```diff
@@ -133,6 +137,10 @@
 	}
 
 	const commitShortSha = commit.id.substring(0, 7);
+	const authorImgUrl =
+		commit.author.email?.toLowerCase() === $user?.email?.toLowerCase()
+			? $user?.picture
+			: commit.author.gravatarUrl;
```

Comment:
Ideally, this should be also a `$derived` value as well. 
Please, check out my comment below.